packages/loader/container-loader/src/protocol.ts

```diff
@@ -124,3 +124,19 @@ export class ProtocolHandler extends ProtocolOpHandler implements IProtocolHandl
 		}
 	}
 }
+
+export function protocolHandlerShouldProcessSignal(message: ISignalMessage) {
+	// Signal originates from server
+	if (message.clientId === null) {
+		const innerContent = message.content as { content: any; type: string };
```

Comment:
Is there a real type/interface we can cast to here instead of an inline type?

Reply:
Sadly, no. These are all somehow undefined bags of "any" type: https://github.com/microsoft/FluidFramework/blob/main/common/lib/protocol-definitions/src/protocol.ts#L317

Reply:
I'd just suggest we use `content: unknown` instead of `content: any`. It's arguably more semantically correct and (I think) less frowned upon by linters and such.